### PR TITLE
Split altinn out in a separate project.

### PR DESCRIPTION
**Pull request by OsirisTerje**
The Altinn code uses Linq.Async which causes ambiguities with Entity Framework.  There is a workaround for this, see the readme file for Altinn, but it seems unnecessary to have this when not using the Altinn code.  So, we're splitting this out in a separate project and thus package, 

Had to change to windows from ubuntu, there is a case issue somewhere in the altinn files, have changed a lot, but it still will not build with ubuntu.

### CODE REVIEW

**Review by robinbaxon — CHANGES_REQUESTED** [review]

File: Fhi.HelseId.Altinn/Fhi.HelseId.Altinn.csproj [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Nullable>enable</Nullable>
+    <LangVersion>8</LangVersion>
+    <Configurations>Debug;Release</Configurations>
+    <id>Fhi.HelseId.Altinn</id>
+    <Version>2.4.0</Version>
```
Comment (robinbaxon) [inline Fhi.HelseId.Altinn/Fhi.HelseId.Altinn.csproj:9]: kanskje bumpe både denne og Fhi:HelseId til 3.0.0? major

File: Fhi.HelseId/Fhi.HelseId.csproj [hunk 2]
```diff
@@ -6,13 +6,13 @@
...
     <id>Fhi.HelseId</id>
-    <Version>2.3.1</Version>
+    <Version>2.4.0</Version>
     <authors>Folkehelseinstituttet (FHI), Nasjonalt Helsenett (NHN)</authors>
     <projectUrl>https://github.com/folkehelseinstituttet/Fhi.HelseId</projectUrl>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <icon>https://github.com/folkehelseinstituttet/Fhi.HelseId/images/fhi.png</icon>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <PackageReleaseNotes>Initial release</PackageReleaseNotes>
+    <PackageReleaseNotes>Split altinn out</PackageReleaseNotes>
```
Comment (robinbaxon) [inline Fhi.HelseId/Fhi.HelseId.csproj:15]: kanskje være litt mer eksplisitt:
"Move Altinn functionality to separate package Fhi.HelseId.Altinn"

File: Fhi.HelseId.Altinn/Subject.cs [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿namespace Fhi.HelseId.Altinn
+﻿namespace Fhi.HelseId.AltInn
```
[the remark this answers is not in the document]
Comment (OsirisTerje) [reply Fhi.HelseId.Altinn/Subject.cs]: og jeg som var 190% sikker på at jeg hadde fikset alle disse........

[the remark this answers is not in the document]
Comment (OsirisTerje) [reply Fhi.HelseId.Altinn/Subject.cs]: og jeg hadde....

File: Fhi.HelseId/Fhi.HelseId.csproj [hunk 4]
```diff
@@ -6,13 +6,13 @@
     <LangVersion>8</LangVersion>
     <Configurations>Debug;Release</Configurations>
     <id>Fhi.HelseId</id>
-    <Version>2.3.1</Version>
+    <Version>2.4.0</Version>
```
[the remark this answers is not in the document]
Comment (OsirisTerje) [reply Fhi.HelseId/Fhi.HelseId.csproj:9]: Done

File: Fhi.HelseId.Altinn/Fhi.HelseId.Altinn.csproj (hunk 1 above)
Comment (OsirisTerje) [reply Fhi.HelseId.Altinn/Fhi.HelseId.Altinn.csproj:9]: Done

File: Fhi.HelseId/Fhi.HelseId.csproj (hunk 2 above)
Comment (OsirisTerje) [reply Fhi.HelseId/Fhi.HelseId.csproj:15]: Done

**Review by robinbaxon — APPROVED** [review]

**Pull request merged**